docs/palette.md

```diff
@@ -2,11 +2,11 @@
 
 Throughout your game you are going to need to use colors in lots of places. There are two classes on `dart:ui` that can be used, `Color` and `Paint`.
 
-The `Color` class is nothing but a wrapper over a single simple ARGB color in hexadecimal integer format, so, to create a Color object, just pass in the color as an integer in the ARGB format.
+The `Color` class is nothing but a wrapper over a single simple ARGB color in hexadecimal integer format. So to create a Color object you just need to pass in the color as an integer in the ARGB format.
 
-You can use Dart's hexadecimal notation to make it really easy; for instance: `0xFF00FF00` is fully opaque green (the 'mask' would be `0xAARRGGBB`). Do note the first two hexadecimal digits are for the alpha channel (transparency), unlike on regular (non-A) RGB. The max(FF = 256) means fully opaque.
+You can use Dart's hexadecimal notation to make it really easy; for instance: `0xFF00FF00` is fully opaque green (the 'mask' would be `0xAARRGGBB`). Do note the first two hexadecimal digits are for the alpha channel (transparency), unlike on regular (non-A) RGB. The max(FF = 255) means fully opaque.
 
-There is a color enum to make it easy to use common colors; it is in the material flutter package:
+There is a color enum to make it easy to use common colors, it resides in the material flutter package:
```

Comment:
I actually think this should still be a `;` :p

Reply:
Hmm, I it works, but it's not really the proper use of the semi-colon in English. It is not perfect with the comma either though. Probably should have rewritten the sentence instead. 😅

Reply:
Fixed!